.github/workflows/backend.yml

```diff
@@ -0,0 +1,103 @@
+name: Backend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'release**'
+      - 'feature**'
+      - 'fix**'
+      - 'ci**'
+    tags:
+      - '**'
+    paths:
+    - 'src/metabase/**'
+    - 'modules/drivers/**'
+    - 'enterprise/backend/**'
+    - '**/project.clj'
+    - '**/deps.edn'
+    - '.github/workflows/**'
```

Comment:
The list of paths to trigger the workflow. Do I miss anything here @camsaul?

Reply:
This is what we have in the CircleCI config:

```yaml
find-args: ". -type f -name '*.clj' -or -name '*.java' -or -name deps.edn -or -name metabase-plugin.yaml"
```

For this I'd change it to be *any* `*.clj` file, `*.edn` file, `*.java` file, or `metabase-plugin.yaml`. There are a few other places Clojure files live like `./backend/mbql` -- it's better just to get everything `*.clj` I think

Reply:
This is great, thanks @camsaul. I'll be adding those patterns, to be on the cautious side.